README.md

```diff
@@ -1,3 +1,6 @@
+# Github Actions Status
+![Build Status](https://github.com/cu-ecen-aeld/assignment-autotest/actions/workflows/github-actions.yml/badge.svg)
```

Comment:
@MuktaDarekar this is the wrong path to use for a repo badge, you should use https://github.com/cu-ecen-aeld/assignment-1-MuktaDarekar/actions/workflows/github-actions.yml/badge.svg

Reply:
Modified it. Done.